### PR TITLE
Feature/7

### PR DESCRIPTION
**Pull request by DPDeepak**
Add a startup method that will set up a DB,
A collection with a sample admin user will be created when there is no user in DB.
This startup method will execute on the initial run of the application.

Ticket:https://github.com/nodexpertsdev/RnD/issues/7

### CODE REVIEW

File: expressEngine/db/seedData.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import DBService from '../lib/service/db.js';
+import { User } from '../model/index.js';
```
Comment (aman-successive) [inline expressEngine/db/seedData.js]: Remove index from import statement

Comment (DPDeepak) [reply expressEngine/db/seedData.js]: removed.

File: expressEngine/db/seedData.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import DBService from '../lib/service/db.js';
+import { User } from '../model';
+
+export default async function seed() {
+  const res = await DBService.count(User);
+
+  if (res <= 0) {
```
Comment (manish-nx) [inline expressEngine/db/seedData.js]: use `if(!res) return;` instead of `if (res <= 0)`

Comment (DPDeepak) [reply expressEngine/db/seedData.js]: Done

**Pull request merged**